README.md

```diff
@@ -100,9 +100,16 @@ And the request throughput of TurboMind is 30% higher than vLLM.
 Install lmdeploy with pip ( python 3.8+) or [from source](./docs/en/build.md)
 
 ```shell
-pip install lmdeploy
+pip install lmdeploy[all]
```

Comment:
can we add a description of `pip install lmdeploy`?

Reply:
sure. updated